deploy-edpm.yml

```diff
@@ -23,6 +23,11 @@
 
 - name: Import package build playbook
   ansible.builtin.import_playbook: ci_framework/playbooks/06-deploy-edpm.yml
+  when: not edpm_compute_baremetal | default(false)
```

Comment:
care to move that `when` one line up, so that it's just bellow the `name` ? it's easier to follow